common/changes/@itwin/core-quantity/Frank-updateEnumsInCore-quantity_2024-10-09-13-45.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-quantity",
+      "comment": "got rid of some blank spaces and refactored the toString methods",
```

Comment:
```suggestion
      "comment": "deprecate FormatEnums toString() methods, convert int enums to string enums",
```

Reply:
changelog files like these are meant to provide information to users and less the contributors - gear the comments in these files to be informative to them.